Sources/DolbyIORTSCore/StreamOrchestrator.swift

```diff
@@ -115,9 +117,20 @@ open class StreamOrchestrator {
                     }
                 }
                 
+                // Unproject audio whose source has been stopped from the publisher
```

Comment:
If we want to `unproject` audio or video when a source is removed - a better place to do that would be in the onInActive call back ?

Reply:
That way we might not need to keep reference of the source whose audio is being played via a `lastSelectedAudioSource`- source itself has a `isPlayingAudio` value  ?

Reply:
Move the things to on inactive